src/components/EndowmentScale.svelte

```diff
@@ -0,0 +1,86 @@
+<script>
+  // Component that draws a rating scale and synchronized pain duration bar to the
+  // screen
+  import { globalVars, calcPropSpent, round2, groupStore } from "../utils.js";
+
+  // INPUTS:
+  export let disabled = false;
+  export let questionText;
+  export let questionType = "other";
+  export let rating;
+  export let endowment;
+
+  $: whoText = questionType === "other" ? "They" : "You";
+  $: otherText = questionType === "other" ? "You" : "They";
+  $: endowmentAmount = endowment;
+
+  let multiplier = globalVars.multiplier;
+
+  let propSpent;
+  $: {
+    const { proportionOfEndowmentSpent } = calcPropSpent(rating, endowment);
+    propSpent = proportionOfEndowmentSpent;
+  }
+  // OUTPUTS:
+  // When using this component use bind:rating={someVariable} to get access to the
+  // rating value
+  // You can also optionally listen for on:change={someFunction} to run someFunction
+  // each time the user changes their rating
+</script>
+
+<div class="flex flex-col px-2 justify-items-center">
+  <p class="mb-2 text-lg">Shared endowment: ${endowment}</p>
+  <p class="mb-2 text-lg">Multiplier: {globalVars.multiplier}</p>
+  <h2 class="my-4 text-2xl">{questionText}</h2>
+</div>
```

Comment:
Can we either remove those or control their visibility with a prop? I like simplifying things with a single rating scale, but it would be nice to have a bit more control over the text that appears above it.

Reply:
ok when you say prop do you mean prop that's input into EndowmentScale?

Reply:
Yea "prop" or "property" is the jargon I've seen used to describe the inputs to web components, like "args" or "parameters" to a function.

Reply:
da9dc2d